### PR TITLE
Clear instruction update

### PR DESCRIPTION
**Pull request by eswari007**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
-# calculator
\ No newline at end of file
+# calculator
+
+This program lets you calculate simple mathematics
```
Comment (harikiran-01) [inline README.md]: instead of mathematics, please use mathematical operations

**Review by harikiran-01 — CHANGES_REQUESTED** [review]
Please address

**Review by harikiran-01 — APPROVED** [review]
LGTM

File: README.md (hunk 1 above)
Comment (eswari007) [reply README.md]: Addressed

**Pull request merged**